org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -3164,15 +3202,37 @@ protected void saveNew_afterSetID()
 	{
 		
 	}
+	
+	/**
+	 * 	Create Single/Multi Key Where Clause
+	 * 	@param withValues if true uses actual values otherwise ?
+	 * 	@return where clause
+	 */
+	public String get_WhereClause (boolean withValues) {
+		return get_WhereClause(withValues,null);
+	}
 
 	/**
 	 * 	Create Single/Multi Key Where Clause
 	 * 	@param withValues if true uses actual values otherwise ?
+	 *  @param uuID RecordUU
```

Comment:
javadoc should be updated to reflect the addtion of uuid option.

Reply:
It is already there. Please note added line is for uuID paramater.